pkg/config/config.go

```diff
@@ -19,6 +19,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/version"
 )
 
+// DefaultForwarderRecoveryInterval is the default recovery interval, also used if
+// the user-provided value is invalid.
+const DefaultForwarderRecoveryInterval = 2
```

Comment:
It would be great to eventually store all of these default forwarder values as globals to avoid duplication/magic numbers.